keyboards/alps64/config.h

```diff
@@ -20,8 +20,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x6464
+#define VENDOR_ID       0x40F3
+#define PRODUCT_ID      0xFBF8
```

Comment:
Where do these value come from?

Reply:
First 4 characters of the shasum for "Hasu" starts with 40F3, first 4 characters of the shasum for "alps64" is FBF8. The default firmware lists 0xFEED and 0x6464 as vendor and product id as seen [here](https://github.com/tmk/tmk_keyboard/blob/master/keyboard/alps64/config.h#L23). As VIA doesn't want 0xFEED as vendor I changed it.

Reply:
I believe the convention is to use two ASCII characters, so "HS" or `0x4853`. If the vendor ID were `0xFEED`, then you would use the shasum for the product ID.